qiskit_experiments/framework/experiment_data.py

```diff
@@ -1147,26 +1147,39 @@ def add_figures(
             if figure_names is None:
                 if isinstance(figure, str):
                     fig_name = figure
-                else:
+                elif not isinstance(figure, FigureData):
```

Comment:
Can you add a comment about the possible types a figure can take, and the expected behavior for each type

Reply:
Done.